image/build.gradle

```diff
@@ -4,4 +4,6 @@ dependencies {
     kapt libs.glide_compiler
 
     api libs.rx_helpers
+    implementation ('com.github.bumptech.glide:okhttp3-integration:4.4.0@aar')
+    implementation 'com.squareup.okhttp3:logging-interceptor:3.6.0'
```

Comment:
Please use `debugImplementation` or similar.

Reply:
Nice catch! Updated

Reply:
huh. forgot we couldn't use that without splitting out code. guess it's fine as-is